hhs_oauth_server/urls.py

```diff
@@ -52,6 +52,20 @@ def robots_txt(request):
     path("v2/o/", include("apps.authorization.v2.urls")),
     path("v3/o/", include("apps.dot_ext.v3.urls")),
     path("v3/o/", include("apps.authorization.v3.urls")),
+    re_path(
```

Comment:
We're currently lacking this for some of the existing v3 endpoints, but we should control access to these with the `v3_endpoints` waffle switch. I believe that can be done either with a function decorator or by providing the waffle switch here in-line, and there should be examples of both forms of that you can reference.